include/aspect/material_model/multicomponent.h

```diff
@@ -188,18 +166,26 @@ namespace aspect
         double reference_T;
 
         /**
-         * Enumeration for selecting which viscosity averaging scheme to use.
-         * Select between harmonic, arithmetic, geometric, and
-         * maximum_composition.  The max composition scheme simply uses the
-         * viscosity of whichever field has the highes volume fraction.
-         */
-        enum
+               * Enumeration for selecting which averaging scheme to use.
+               * Select between harmonic, arithmetic, geometric, and
+               * maximum_composition.  The max composition scheme simply uses the
+               * parameter of whichever field has the highest volume fraction.
+               */
+        enum averaging_scheme
```

Comment:
Also, since `averaging_scheme` is a type, please name it `AveragingScheme` (we use CamelCase for type names).